_posts/0000-01-02-Lipserver.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
This seems madening... Review Changes - where is the complaint?  What do I need to edit?  Why?  Why Line 5?